ambassador/ambassador/ir/irmapping.py

```diff
@@ -427,8 +429,8 @@ def finalize(self, ir: 'IR', aconf: Config) -> List[IRCluster]:
         """
         Finalize a MappingGroup based on the attributes of its Mappings. Core elements get lifted into
         the Group so we can more easily build Envoy routes; host-redirect and shadow get handled, etc.
-        
-        :param ir: the IR we're working from 
+
+        :param ir: the IR we're working from
```

Comment:
Can you elaborate it a bit more, please?

Reply:
This was also just a whitespace change.